### PR TITLE
Fixed credit?

### LINKED ISSUE #17: Add to CREDITS.md
Add this url with the thank you note in [CREDITS.md](https://github.com/arshadkazmi42/arshadkazmi42.github.io/blob/develop/CREDITS.md)

`https://dribbble.com/shots/2316666-Profile-card-UI`

This is designer's profile 
https://dribbble.com/sokratus

You can get name of the designer also from the above url.

UI of this application is inspired from this.

### PR DESCRIPTION
**Pull request by Killernotch101**
Fixes #17 
---------
Please tell me if you would like anything added, or changed. (Does it need "Licensed by"?)

### CODE REVIEW

File: CREDITS.md [hunk 1]
```diff
@@ -3,3 +3,4 @@
 <div>Icons made by <a href="http://www.freepik.com" title="Freepik">Freepik</a> from <a href="https://www.flaticon.com/" title="Flaticon">www.flaticon.com</a> is licensed by <a href="http://creativecommons.org/licenses/by/3.0/" title="Creative Commons BY 3.0" target="_blank">CC 3.0 BY</a></div>
 <div>Icons made by <a href="https://www.flaticon.com/authors/dave-gandy" title="Dave Gandy">Dave Gandy</a> from <a href="https://www.flaticon.com/" title="Flaticon">www.flaticon.com</a> is licensed by <a href="http://creativecommons.org/licenses/by/3.0/" title="Creative Commons BY 3.0" target="_blank">CC 3.0 BY</a></div>
 <div>Icons made by <a href="https://www.flaticon.com/authors/vectors-market" title="Vectors Market">Vectors Market</a> from <a href="https://www.flaticon.com/" title="Flaticon">www.flaticon.com</a> is licensed by <a href="http://creativecommons.org/licenses/by/3.0/" title="Creative Commons BY 3.0" target="_blank">CC 3.0 BY</a></div>
+<div>The UI has been inspired by <a href="https://dribbble.com/sokratus" title="dribble">Sokratus</a> from <a href="https://dribble.com/" title="dribble">Dribble</a>.</div>
```
Comment (arshadkazmi42) [inline CREDITS.md]: Can you also add the URL of the UI.

**Comment by arshadkazmi42** [comment]
Also, add the issue number in the Pull request comments.

**Comment by Killernotch101** [comment]
#17 is the issue.

File: CREDITS.md (hunk 1 above)
Comment (Killernotch101) [reply CREDITS.md]: added!

File: CREDITS.md [hunk 2]
```diff
@@ -3,3 +3,4 @@
...
 <div>Icons made by <a href="https://www.flaticon.com/authors/dave-gandy" title="Dave Gandy">Dave Gandy</a> from <a href="https://www.flaticon.com/" title="Flaticon">www.flaticon.com</a> is licensed by <a href="http://creativecommons.org/licenses/by/3.0/" title="Creative Commons BY 3.0" target="_blank">CC 3.0 BY</a></div>
 <div>Icons made by <a href="https://www.flaticon.com/authors/vectors-market" title="Vectors Market">Vectors Market</a> from <a href="https://www.flaticon.com/" title="Flaticon">www.flaticon.com</a> is licensed by <a href="http://creativecommons.org/licenses/by/3.0/" title="Creative Commons BY 3.0" target="_blank">CC 3.0 BY</a></div>
+<div><a href="https://dribbble.com/shots/2316666-Profile-card-UI" title="UI">The UI</a> has been inspired by <a href="https://dribbble.com/sokratus" title="dribble">Sokratus</a> from <a href="https://dribble.com/" title="dribble">Dribble</a>.</div>
```
Comment (arshadkazmi42) [inline CREDITS.md]: Lets change it to something like this

`<div>The UI has been inspired by <a href="https://dribbble.com/sokratus" title="dribble">Sokratus's</a> <a href="https://dribbble.com/shots/2316666-Profile-card-UI" title="UI">work</a> from <a href="https://dribble.com/" title="dribble">Dribble</a>.</div>`

How does this looks? Any thoughts about this?

**Comment by arshadkazmi42** [comment]
This has been fixed at #30

**Pull request closed without merging**